src/components/LandingPage/style.css

```diff
@@ -85,7 +85,6 @@
 
 .markdown {
     column-count: 2;
-    height: 540px;
```

Comment:
this is the change that fixes the problem (see screen shots for before and after of the acknowledgments)

Reply:
That's what I would have done (I had made this issue for this bug but looks like it got buried - https://github.com/simularium/simularium-website/issues/329)

Reply:
Oh man! I totally missed this, I'll be better about searching in the issues in the future. Thanks for reviewing!